liquidprompt

```diff
@@ -1133,6 +1134,31 @@ _lp_python_env_color() {
     lp_python_env_color="[${LP_COLOR_VIRTUALENV}${lp_python_env}${NO_COL}]"
 }
 
+_lp_ruby_env() {
+    (( LP_ENABLE_RUBY_VENV )) || return 2
+
+    local ret
+
+    if command -v rvm-prompt >/dev/null ; then
+        [ -z "${LP_RUBY_RVM_PROMPT_OPTIONS-}" ] && LP_RUBY_RVM_PROMPT_OPTIONS=(i v g s)
+        __lp_escape "$(rvm-prompt $LP_RUBY_RVM_PROMPT_OPTIONS)"
+        lp_ruby_env=$ret
+    elif command -v rbenv >/dev/null ; then
+        local rbenv_ver=$(rbenv version)
```

Comment:
If doing `local` and assignment on the same line, Zsh 5.0.X will break without quotes (see [Shell Standards Wiki page](https://github.com/nojhan/liquidprompt/wiki/Shell-standards)).
```suggestion
        local rbenv_ver="$(rbenv version)"
```